rose/web/index.html

```diff
@@ -36,20 +36,19 @@
           <div class="name"></div>
           <div class="score"></div>
         </div>
-        </div>
+      </div>
+
       <canvas id="game" width="910" height="585">
         <h2>Unsupported browser</h2>
       </canvas>
-
     </div>
 
-    <script src="jquery-3.2.1.min.js"></script>
     <script src="game.js"></script>
```

Comment:
ok, as the code is located at the end of the file, and we know `ready()`is the only script that need to run, it's ok to femove the `DOMContentLoaded` ...

I'll update the code to use import/export  and remove the wait for DOMContentLoaded

Reply:
No because modules aren't executed until the DOM is constructed

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules#other_differences_between_modules_and_standard_scripts
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#defer

Reply:
nice :fire: 

added a separate commit for the module update, left it in the end for now :-)